low-level/framework/utils/conf_utils.py

```diff
@@ -151,7 +150,6 @@
 Conf.load(SSPL_CONF, "yaml:///etc/sspl.conf")
 global_config = Conf.get(SSPL_CONF, "SYSTEM_INFORMATION>global_config_copy_url")
 Conf.load(GLOBAL_CONF, global_config)
-Conf.load(CLUSTER_CONF, "json:///etc/cortx/cluster.conf")
```

Comment:
why removing ? CLUSTER_CONF supposed to be different from GLOBAL_CONF ! or is it merged now ?

Reply:
I am getting the same CVG related information in the global conf of the SSPL.
We are coping with this at the time of the SSPL deployment. So I feel we can refer SSPL Global file itself and we have already loaded it in conf_utils.py
> /etc/sspl_global_config_copy.yaml

Reply:
No, its just not about CVG, we may be depending on more things from cluster.conf, see what all code using it. @mariyappanp any thoughts?

Reply:
[6:58 PM] Satish Darade
    


Conf.load(CLUSTER_CONF, "json:///etc/cortx/cluster.conf")

Its CORTX cluster level config, deployed by Provisioner, which components like SSPL uses to copy & populate its own config during setup, like sspl_global_config_copy.yaml.

I know run time we wont need it, i just wanted to make sure to check for any references and confirm


Chetan, This code I have introduced in my last PR (https://github.com/Seagate/cortx-monitor/pull/509/files) and after mariyappan pr comments, I have checked the same information is available in sspl global config also. SO In this PR, I have removed those added codes from my last pr. (https://github.com/Seagate/cortx-monitor/pull/509/files)
<https://teams.microsoft.com/l/message/19:95e8d89deb2a4213921cfa821f652e4f@thread.tacv2/1630934909150?tenantId=d466216a-c643-434a-9c2e-057448c17cbe&amp;groupId=7e2a2869-5847-4a65-9402-f4778c7b913e&amp;parentMessageId=1630314433839&amp;teamName=CORTX.All&amp;channelName=EOS-RAS&amp;createdTime=1630934909150>


[7:00 PM] Mariyappan Ponnusamy
    

cortx-utils creates /etc/cortx/cluster.conf using provisioner input config.

Modifying or referring cluster.conf directly may interrupt other process. So we can refer sspl_global_config.yaml instead of that. Both will have same info.

Edited<https://teams.microsoft.com/l/message/19:95e8d89deb2a4213921cfa821f652e4f@thread.tacv2/1630935057677?tenantId=d466216a-c643-434a-9c2e-057448c17cbe&amp;groupId=7e2a2869-5847-4a65-9402-f4778c7b913e&amp;parentMessageId=1630314433839&amp;teamName=CORTX.All&amp;channelName=EOS-RAS&amp;createdTime=1630935057677>